.travis.yml

```diff
@@ -33,6 +34,8 @@ matrix:
           sources: ubuntu-toolchain-r-test
           packages: ninja-build g++-5
       env: DXC_BUILD_TYPE=Release
+  allow_failures:
```

Comment:
We don't need this anymore?

Reply:
The mac tests are passing. So we don't need it unless you want to let them fail in the future without impeding submissions.